packages/get-platform/src/getPlatform.ts

```diff
@@ -289,10 +390,11 @@ ${additionalMessage}`,
 
   // sometimes we fail to detect the distro in use, so we default to debian
   const defaultDistro = 'debian' as const
-  if (platform === 'linux' && distro === undefined) {
+  if (platform === 'linux' && targetDistro === undefined) {
     warnOnce(
       'distro:undefined',
-      `Prisma failed to detect the Linux distro in use, and may not work as expected. Defaulting to "${defaultDistro}".`,
+      `Prisma doesn't know which engines to download for the Linux distro "${originalDistro}". Falling back to Prisma engines built for "${defaultDistro}". While these engines are likely to work on your system, Prisma may not work as expected.
+Please run \`prisma generate\` and \`prisma db push\` to chat that Prisma is working correctly on your system.`,
```

Comment:
"chat"?

Reply:
These commands are really bad ideas to just run as well. Especially `db push` can completely mess with a database.

This will be very common output for some users (all on not explicitly supported Linux distros).
We should make it shorter, not even longer and more complex.

Reply:
Whelp, "chat" was supposed to be "check"

Reply:
I agree to have a link to the docs as it can be a nice place to store a specific how-to/tutorial (eg. building your own engines, environment variables, etc...).